db/migrate/20220319025757_create_area_vets.rb

```diff
@@ -0,0 +1,12 @@
+class CreateAreaVets < ActiveRecord::Migration[7.0]
```

Comment:
Drop last migration (rollback) and change this one so the model is properly created